internal/cmd/logs-files-metric-exporter/symlink-logfiles-watcher.go

```diff
@@ -0,0 +1,131 @@
+package main
```

Comment:
@alanconway please chime in on the name.  It still seems a little off with multiple pluralization.  Should it be ?:
* log-file-metric-exporter
* log-files-metric-exporter